### PR TITLE
Add script to tail logfiles from the VCH [skip ci]

### PR DESCRIPTION
**Pull request by fdawg4l**
`./tail.sh <hostname or ip> <docker.log | port-layer.log>` to tail the log file from your dev env.

### CODE REVIEW

File: infra/scripts/tail.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -e
+# Copyright 2016 VMware, Inc. All Rights Reserved.
```
Comment (mdubya66) [inline infra/scripts/tail.sh]: 2017

Comment (fdawg4l) [inline infra/scripts/tail.sh]: Thanks sophie-bot!

File: infra/scripts/tail.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    # extract password from $GOVC_URL
+    password=$(govc env | grep GOVC_PASSWORD= | cut -d= -f 2-)
+fi
+
+if [ -z "$username" ] ; then
+    username=$(govc env | grep GOVC_USERNAME= | cut -d= -f 2-)
+fi
+
+jar=$(mktemp -t cookie-XXXX)
+
```
Comment (dougm) [inline infra/scripts/tail.sh]: could use `trap 'rm $jar' EXIT`, so the temp dir is removed even if commands below fail.

**Comment by dougm** [comment]
Also, bonus points if you make the script shellcheck clean.

**Comment by hickeng** [comment]
You could also add a vic-tail function to the bash-helpers - if there's a client certificate then you can go via vicadmin without doing the ssh dance by providing that to curl.

File: infra/scripts/vic-logs.sh [hunk 3]
```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -e
+# Copyright 2016 VMware, Inc. All Rights Reserved.
```
Comment (mhagen-vmware) [inline infra/scripts/vic-logs.sh]: 2017

**Pull request merged**